thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/FunctionOnboardingResource.java

```diff
@@ -0,0 +1,41 @@
+package com.linkedin.thirdeye.anomaly.onboard;
+
+import com.linkedin.thirdeye.datalayer.bao.AnomalyFunctionManager;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import com.linkedin.thirdeye.datasource.DAORegistry;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+
+
+@Path("/detection-onboard")
```

Comment:
this namespace clashes with the existing "detection-onboard" resource and/or the reverse proxy. Please choose a different name.

Reply:
Change to function-onboard.